### PR TITLE
remove deprecated parameters

### PR DESCRIPTION
**Pull request by inkhey**
related to #1483 

## Checkpoints

These points must be checked before merging. Please don't edit them out.

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link) :https://github.com/tracim/tracim/pull/3471#issuecomment-683742889

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/doc/setting.md [hunk 1]
```diff
@@ -62,125 +59,10 @@ The priority order is (from less to more priority):
...
-| TRACIM_EMAIL__NOTIFICATION__UPLOAD_PERMISSION_TO_EMITTER__TEMPLATE__HTML  | email.notification.upload_permission_to_emitter.template.html  | EMAIL__NOTIFICATION__UPLOAD_PERMISSION_TO_EMITTER__TEMPLATE__HTML  |
-| TRACIM_EMAIL__NOTIFICATION__UPLOAD_PERMISSION_TO_EMITTER__SUBJECT         | email.notification.upload_permission_to_emitter.subject        | EMAIL__NOTIFICATION__UPLOAD_PERMISSION_TO_EMITTER__SUBJECT         |
-| TRACIM_EMAIL__NOTIFICATION__NEW_UPLOAD_EVENT__TEMPLATE__HTML              | email.notification.new_upload_event.template.html              | EMAIL__NOTIFICATION__NEW_UPLOAD_EVENT__TEMPLATE__HTML              |
-| TRACIM_EMAIL__NOTIFICATION__NEW_UPLOAD_EVENT__SUBJECT                     | email.notification.new_upload_event.subject                    | EMAIL__NOTIFICATION__NEW_UPLOAD_EVENT__SUBJECT                     |
-| <env_var_name>                                                            | <config_file_name>                                             | <config_name>                                                      |
```
Comment (inkhey) [inline backend/doc/setting.md:135]: The current documentation have duplication of config parameter, i do remove this.

**Comment by inkhey** [comment]
Automatic test for this doesn't make any sense. This is just removing dead code or untested fallback code.

File: backend/doc/setting.md [hunk 2]
```diff
@@ -14,10 +14,10 @@ The priority order is (from less to more priority):
 - configuration file
 - environnement variables
 
+<!--- Maintainer: use tracimcli dev parameters list --template "| {env_var_name: <74}| {config_file_name: <63}|{config_name: <68}|"  with all app enabled to update this list properly --->
```
Comment (raphj) [inline backend/doc/setting.md:17]:
```suggestion
<!--- Maintainer: use tracimcli dev parameters list --template "| {env_var_name: <74}| {config_file_name: <63}|{config_name: <68}|"  with all apps enabled to update this list properly --->
```

Why put an important information in an HTML comment that will not be rendered?

Comment (inkhey) [reply backend/doc/setting.md:17]: The problem here is : 
- Document how to update the markdown file
- Distinct clearly what is backside setup (maintaining the list to have up to date documentation) and the documentation.
If you have a better suggestion, i'm listening.

Comment (inkhey) [reply backend/doc/setting.md:17]: if you are not maintaining the documentation there is no new to see this.

Comment (raphj) [reply backend/doc/setting.md:17]: This distinction must not be done by hiding things on the rendered document I think. If you need to split things, maybe use a different section in the file or a different file?

Comment (inkhey) [reply backend/doc/setting.md:17]: If you put this anywhere else, someone will not see it and will forgot it.

Comment (raphj) [reply backend/doc/setting.md]: Okay, but if you hide it in a comment, people looking at the rendered document will not see it. I can't manage to follow you here. @grignards?

Comment (inkhey) [reply backend/doc/setting.md]: note: there is a documentation about the command line use here, but it's more generic, this comment is really about maintaining properly the list just here in this documentation. The command here is already documented somewhere else.

Comment (raphj) [reply backend/doc/setting.md]: Oh, I see. This comment is intented at people editing this file. Still, they could very well think of editing the document before opening it.
I don't know what is best. Another opinion would not be too much I think.

Comment (grignards) [reply backend/doc/setting.md]: I do like the idea of putting the instructions in the document which is better than having no precise instructions of how to update it. Even better would be to change the `dev parameters list` command in order to integrate this "standard" template directly as an option/command, but this is not part of this issue.

**Review by grignards — APPROVED** [review]

**Review by raphj — APPROVED** [review]

**Comment by gbonaspetti** [comment]
Tested on branch.

**Pull request merged**